File: python/iniciante/1043.py
'''
title: Triângulo
number of the problem: 1043
date: 04/03/2018
author: ltsdw
'''

a, b, c = map(float, input('').split())

if a + b > c and c + a > b and c + b > a:
    print(f'Perimetro = {a + b + c:.1f}')
else:
    print(f'Area = {(a + b) * c / 2:.1f}')

File: haskell/iniciante/1043.hs
-- title: Triângulo
-- number of the problem: 1043
-- date: 22/09/20
-- author: ltsdw

import Text.Printf

f1 a b c = a + b + c
f2 a b c = (a + b) * c / 2.0

main :: IO ()
main = do
    l1 <- getLine

    let (a : b : c : _) = map (read :: String -> Double) $ words l1

    if ((a + b) > c) && ((c + a) > b) && ((c + b) > a)
        then do
            printf "Perimetro = %.1f\n" ( f1 a b c :: Double )
    else
        printf "Area = %.1f\n" ( f2 a b c :: Double ) 

